src/hooks/queries/useHomeGalleryAssets.js

```diff
@@ -38,7 +39,55 @@ const useHomeGalleryAssets = () => {
     }
   `);
 
+  let LARGE_PICTURE = {};
+  let MEDIUM_PICTURE = {};
+  let SMALL_PICTURE = {};
+
+  const { breakpoint, width } = useBreakpoints({
+    breakpoints: {
```

Comment:
breakpoints naming isn't very clear. 
let's change it to: 
```
breakpoints: {
      desktop: 1280,
      mobile: 0,
    },
```

I talked to @Aleksandr-Tyagun , it'll work the same way as 
```
desktopLg: 1280,
desktop: 768,
```